odra-casper/casper-client/Cargo.toml

```diff
@@ -19,5 +19,5 @@ itertools = "0.10.5"
 blake2 = { version = "0.9.0", default-features = false }
 dotenv = "0.15.0"
 prettycli = "0.1.1"
-thiserror = "1.0.40"
+thiserror = "1.0.34"
```

Comment:
Why downgraded?

Reply:
It was required by `casper-client` but is not anymore.